gulpfile.js

```diff
@@ -123,6 +123,7 @@ gulp.task('manifest:production', function() {
   ],{base: './dist/'})
   .pipe(gulpif(!debug,jsoneditor(function(json) {
     json.background.scripts = ["scripts/background.js"]
+    json["content_scripts"][1]["exclude_matches"] = [`${MASCARA_ORIGIN}*`]
```

Comment:
content_scripts is an array with an object already in it. The object already in it is the one we want to write too